### PR TITLE
Feature: change main colors palette

### PR DESCRIPTION
**Pull request by oyershov**

### CODE REVIEW

File: src/styles/themes/light.pcss [hunk 1]
```diff
@@ -22,7 +22,8 @@
   --primary-text-color: #171C29;
   --text-contrast-color: #FFFFFF;
   --secondary-text-color: #28334E;
-  --icons: #737F92;
+  --icons: #28334E;
+  --icons-contrast-color: #28334E;
```
Comment (josadcha) [inline src/styles/themes/light.pcss]: just a small remark to define icon contrast color different from icons color

**Comment by pkucherenk0** [comment]
1. Missing borders on beneficiary 
Selection_113
Selection_112

2. KYC highlight incorrect input and correct input have same color
Selection_115

**Comment by pkucherenk0** [comment]
Background of step numbers are not visible now.
Selection_120

File: src/screens/ChangeForgottenPasswordScreen/ChangeForgottenPasswordScreen.pcss [hunk 2]
```diff
@@ -16,12 +16,12 @@
       min-height: calc(var(--big-gap) * 5.5);
 
       &__option {
-        background: var(--header);
+        background: var(--header-background-color);
       }
       &__options-group {
```
Comment (akhlopiachyi) [inline src/screens/ChangeForgottenPasswordScreen/ChangeForgottenPasswordScreen.pcss]: add empty line before

File: src/screens/ConfirmScreen/ConfirmScreen.pcss [hunk 3]
```diff
@@ -161,41 +159,41 @@
...
+          color: var(--primary-text-color);
         }
       }
        .pg-confirm__active-2 {
-        color: var(--color-white);
+        color: var(--primary-text-color);
         font-weight: bold;
       }
     }
     &-third {
```
Comment (akhlopiachyi) [inline src/screens/ConfirmScreen/ConfirmScreen.pcss]: add empty line before

File: src/screens/SignInScreen/SignInScreen.pcss [hunk 4]
```diff
@@ -63,14 +63,14 @@
     }
 
     &__forgot {
-      color: var(--base-new);
+      color: var(--secondary-text-color);
       padding-top: calc(var(--gap) * 1.33);
     }
     &__input {
```
Comment (akhlopiachyi) [inline src/screens/SignInScreen/SignInScreen.pcss]: add empty line before

File: src/screens/SignInScreen/SignInScreen.pcss [hunk 5]
```diff
@@ -88,18 +88,18 @@
         font-weight: bolder;
       }
       &-signup {
-        color: var(--active-color);
+        color: var(--primary-cta-color);
         font-size: calc(var(--gap) * 2.33);
         font-weight: bold;
       }
     }
     &__group--focused {
```
Comment (akhlopiachyi) [inline src/screens/SignInScreen/SignInScreen.pcss]: add empty line before

File: src/screens/SignInScreen/SignInScreen.pcss [hunk 6]
```diff
@@ -88,18 +88,18 @@
         font-weight: bolder;
       }
       &-signup {
```
Comment (akhlopiachyi) [inline src/screens/SignInScreen/SignInScreen.pcss]: add empty line before

File: src/screens/SignUpScreen/SignUpScreen.pcss [hunk 7]
```diff
@@ -95,7 +95,7 @@
         font-weight: bolder;
       }
       &-signin {
```
Comment (akhlopiachyi) [inline src/screens/SignUpScreen/SignUpScreen.pcss]: add empty line before

File: src/screens/WalletsScreen/WalletsScreen.pcss [hunk 8]
```diff
@@ -53,17 +53,17 @@
       padding: 0 !important;
       &:last-child {
         .cr-wallet-item {
-          border-bottom: 0.5px solid var(--wallets-item-border);
+          border-bottom: 0.5px solid var(--divider-color);
         }
       }
```
Comment (akhlopiachyi) [inline src/screens/WalletsScreen/WalletsScreen.pcss]: remove empty lines

File: src/screens/WalletsScreen/WalletsScreen.pcss [hunk 9]
```diff
@@ -526,12 +525,12 @@
...
       &__body {
-        background-color: var(--background-dark);
+        background-color: var(--body-background-color);
         tr {
```
Comment (akhlopiachyi) [inline src/screens/WalletsScreen/WalletsScreen.pcss]: add empty line before

File: src/screens/WalletsScreen/WalletsScreen.pcss [hunk 10]
```diff
@@ -526,12 +525,12 @@
...
-          background-color: var(--background-dark);
-          border-bottom: 1px solid var(--header-border);
+          background-color: var(--body-background-color);
+          border-bottom: 1px solid var(--divider-color);
           td {
```
Comment (akhlopiachyi) [inline src/screens/WalletsScreen/WalletsScreen.pcss]: add empty line before

File: src/styles/colors.pcss [hunk 11]
```diff
@@ -1,144 +1,20 @@
...
-  --trading-header-fastsearch-button-border-active: var(--accent);
-  --trading-header-positive: var(--green);
-  --trading-header-negative: var(--red);
-  --trading-header-search-border: var(--header-border);
+  --trading-header-fastsearch-button-color-active: var(--primary-text-color);
 
   /* Tradingview */
-  --trading-view-background: var(--main-bg);
+  --trading-view-background: var(--body-background-color);
 }
```
Comment (akhlopiachyi) [inline src/styles/colors.pcss]: add empty line in the end of file

File: src/styles/themes/light.pcss [hunk 12]
```diff
@@ -1,69 +1,66 @@
...
   --text-muted: rgba(#28334E, 0.5);
-  --table-header-color: var(--bright);
+  --input-border: rgba(#28334E, 0.3);
+
+  /* Call to action (CTA) color opacity */ 
+  --cta-color-opacity-48: rgba(#3C78E0, 0.48);
+
+  /* Call to action (CTA) contrast opacity */  
+  --cta-contrast-opacity-48: rgba(#FFF, 0.48);
+  --cta-contrast-opacity-0-1: rgba(#3C78E0, 0.1);
```
Comment (akhlopiachyi) [inline src/styles/themes/light.pcss]: name of this value is totally wrong

--cta-contrast-opacity-12 is correct one

Please, check all values in this file and do by system -12, -24, -36, so on

**Comment by akhlopiachyi** [comment]
This pull request seems uncompleted. Contact designers for better describing this tasks about color

**Comment by akhlopiachyi** [comment]
You have to split this pr to 2
1. Change main colors palette
2. Convert svg icons to tsx

**Pull request closed without merging**